islandora-http-client/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<blueprint xmlns="http://www.osgi.org/xmlns/blueprint/v1.0.0"
+       xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xmlns:cm="http://aries.apache.org/blueprint/xmlns/blueprint-cm/v1.1.0"
+       xsi:schemaLocation="
+       http://aries.apache.org/blueprint/xmlns/blueprint-cm/v1.1.0 http://aries.apache.org/schemas/blueprint-cm/blueprint-cm-1.1.0.xsd
+       http://www.osgi.org/xmlns/blueprint/v1.0.0 http://www.osgi.org/xmlns/blueprint/v1.0.0/blueprint.xsd">
+
+  <!-- OSGI blueprint property placeholder -->
+  <cm:property-placeholder id="properties" persistent-id="ca.islandora.alpaca.http.client" update-strategy="reload" >
+    <cm:default-properties>
+      <cm:property name="token.value" value="islandora"/>
+    </cm:default-properties>
+  </cm:property-placeholder>
+  
+  <service id="httpClient" interface="org.apache.http.client.HttpClient" filter="(osgi.jndi.service.name=claw/httpClient)">
```

Comment:
Does this work?  I've never tried the `filter` property when _exporting_ a service.  My only experience is with the form:

    <service-properties>
      <entry key="osgi.jndi.service.name" value="claw/httpClient" />
    </service-properties>